RetailSuiteAccounting/Operation/06_DeleteBookings.md

```diff
@@ -0,0 +1,72 @@
+[!!User Interface bookings](../UserInterface/01a_Bookings.md)  
+[!!User Interface delete](../UserInterface/01_Book.md#delete)  
+
+
+# Delete bookings
+
+The *Accounting* module allows to delete bookings in the *BOOKINGS* tab, for example, if content errors are found or if the booking was created for test purposes.
+
+It is possible to delete single bookings or multiple bookings at once. Bear in mind, however, that the bookings to be deleted must be unprocessed. For detailed information, see [Process bookings](./07_ProcessBookings.md). In fact, a booking that has already been processed cannot be deleted, as the [DELETE] button will change to inactive (grayed out) when clicking on a unprocessed booking.
+
+## Delete a single booking
+
+#### Prerequisites
+
+- The accounting is configured via the configuration wizard, see [Run the accounting wizard](../Integration/01_RunAccountingWizard.md).
+- A valid fiscal year is created, see [Create a fiscal year](../Integration/04_ManageFiscalYear.md#create-a-fiscal-year).
+- A fiscal year is selected, see [Select fiscal year](./01_SelectFiscalYear.md).
+- A booking has been created, see [Create a manual booking](./04_CreateManualBooking.md).
+
+#### Procedure
+
+*Accounting > Select fiscal year > Book > Tab BOOKINGS*
+
+![Delete booking](../../Assets/Screenshots/RetailSuiteAccounting/Book/Bookings/DeleteBooking.png "[Delete booking]")
+
+1. Select the booking to be deleted from the list in the *BOOKINGS* tab.
+
+2. The existing booking details are displayed in the input line fields.
+
+3. Click the [DELETE] button.  
+A confirmation window is displayed.
+
+  > [Warning] Be aware that the existing booking will be deleted permanently.
+
+4. Click the [Ok] button to confirm the action.  
+A message above the input line confirms that the booking has been deleted. The deleted booking will no longer be displayed in the bookings list.
+
+  ![Booking deleted](../../Assets/Screenshots/RetailSuiteAccounting/Book/Bookings/BookingDeleted.png "[Booking deleted]")
+
+
+## Delete multiple bookings
+
+#### Prerequisites
+
+- The accounting is configured via the configuration wizard, see [Run the accounting wizard](../Integration/01_RunAccountingWizard.md).
+- A valid fiscal year is created, see [Create a fiscal year](../Integration/04_ManageFiscalYear.md#create-a-fiscal-year).
+- A fiscal year is selected, see [Select fiscal year](./01_SelectFiscalYear.md).
+- Multiple bookings have been created, see [Create a manual booking](./04_CreateManualBooking.md).
+
+#### Procedure
+
+*Accounting > Select fiscal year > Book > Tab BOOKINGS > Context menu EDIT > Menu entry Delete*
+
+![Delete multiple bookings](../../Assets/Screenshots/RetailSuiteAccounting/Book/Bookings/DeleteMultipleBookings.png "[Delete multiple bookings]")
+
```

Comment:
1. Click the [EDIT] button in the header above the postings list.
   A context menu is displayed.

Bild Kontextmenü einfügen